### DISCUSSION TITLE
125284e2-9963-80d0-b12d-f9bb320cb453

### DISCUSSION THREAD

**Discussion by giscus** [varofla/blog_giscus#27] [General]
# 125284e2-9963-80d0-b12d-f9bb320cb453

진짜 IoT

https://www.varofla.com/125284e2-9963-80d0-b12d-f9bb320cb453

**Comment by zxcvkim** [comment]
IoT 관련해서 검색하다가 여기까지 오게 되었는데 혹시 BTHome을 잘 아시나요??

**Reply by varofla** [reply]
안녕하세요 :wave:

아쉽게도 처음 들어봅니다.
찾아보니 HomeAssistant의 에드온 중 BLE판 Z2M같은 느낌인것 같네요.
BLE 장치를 HA에 연결할 때 유용할 것 같습니다.
https://bthome.io/format/
